javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,41 @@
+const class07Students = [];
+const queenName = 'Margrethe II';
+function addStudentToClass(studentName) {
+  if (studentName === '') {
+    console.log('Please enter a name of a student');
+  } else if (class07Students.includes(studentName)) {
+    console.log('Student ' + studentName + ' is already in the class');
+  } else if (class07Students.length <= 5 || studentName === queenName) {
```

Comment:
class07Students.length <= 6

Reply:
After checking this condition we add a new student to the class. The max capacity of the class is 6 students, so if it is <= 5 we can add the sixth student, and if it would be <=6 - it would be possible to add the seventh student.

Reply:
right :)

Reply:
your right :)